crates/vizia_core/src/context/mod.rs

```diff
@@ -115,7 +130,10 @@ impl Context {
             focus_stack: Vec::new(),
             cursor_icon_locked: false,
             resource_manager: ResourceManager::new(),
-            text_context: TextContext::default(),
+            cosmic_context: CosmicContext::new_from_locale_and_db(
+                sys_locale::get_locale().unwrap_or_else(|| "en-US".to_owned()),
+                db,
+            ),
```

Comment:
Didn't realise you need a locale to initialise the cosmic context. I guess this means we should probably try to provide it with the system locale, which you can get with the `sys_locale` crate (see `environment.rs` for example).

Reply:
Is that not what's being done right now?

Reply:
Haha, yep, I can't read. I saw the fallback to en-US and thought that was being passed in.